### ISSUE TITLE
Better document logging

### ISSUE THREAD

**Issue by nichtich** [nichtich/Plack-App-GitHub-WebHook#15]
e.g. to a file

**Comment by nichtich** [comment]
Support 

```perl
Plack::App::GitHub::WebHook->new( logger => $logger )
``` 

to facilitate

```perl
 builder {
        enable sub {
            my $app = shift;
            sub {
                my $env = shift;
                $env->{'psgix.logger'} =  $logger;
                $app->($env);
            }
        },
        Plack::App::GitHub::WebHook->new(...)
}
```

**Comment by nichtich** [comment]
implemented in 0.8.1 to be released in a moment.
